packages/workbox-routing/Router.mjs

```diff
@@ -77,15 +80,10 @@ class Router {
       return;
     }
 
-    let route = null;
-    let handler = null;
-    let params = null;
-    let debugMessages = [];
+    let {params, route} = this.findMatchingRoute({url, request, event});
```

Comment:
Looks like all these `let`s can be changed to `const` after your refactoring.

Reply:
`route` gets reassigned and `params` doesn't, but since one of them does, in order to use destructuring I had to do both as let...